docs/en/seatunnel-engine/deployment.md

```diff
@@ -179,6 +179,7 @@ map:
            type: hdfs
            namespace: /tmp/seatunnel/imap
            clusterName: seatunnel-cluster
+           storage.type: hdfs
```

Comment:
Is there a default value? for older versions

Reply:
> Is there a default value? for older versions

default hdfs